code/src/index.css

```diff
@@ -5,9 +5,180 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(0, 0, 0);
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  color:white;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-bottom: 1px solid #65686e;
+  font-size: 27px;
+}
+
+h3 {
+  font-size: 15px;
+  color: #ffffff;
+  margin-bottom: 5px;
+}
+
+h3:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+a {
+  color: rgb(168, 168, 168);
+  text-decoration: none;
+  font-size: 15px;
+  
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+/* Album */
+
+.albums-container {
+  width: 87%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: auto;
+}
+
+.album-card {
+  margin: 20px;
+}
+
+.album-artists {
+  color: white;
+}
+
+/*To hide the last comma after the artist name*/
+.album-artists a:last-child span {
+  display: none;
+}
+
+.album-cover {
+  position: relative;
+  width: 50%;
+}
+
+/* Icons */
+
+.icons {
+  position: absolute;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  top: 50%;
+  left: 82%;
+  display: none;
+  transform: translate(-50%,-50%);
+  z-index: 5;
+  width: 90%;
+}
```

Comment:
Like you mentioned on our Monday meeting, there are some issued with the display of the icons. They are not centered, and when you use a very large screen they move outside of the album cover img. We're not really sure exactly what's wrong, but we find it peculiar that the left percentage is so high (our solution is "left 4%"). Also, you probably should not use two "display" in the same {}. This probably explains why your "justify-content" and "align-items" do not really work.